src/collectors/windows.plugin/perflib-network.c

```diff
@@ -137,6 +223,107 @@ struct network_protocol {
         },
     },
 
+    {
+        .protocol = "IPv4",
+        .packets = {
+            .InDiscards = { .key = "Datagrams Received Discarded" },
+            .OutDiscards = { .key = "Datagrams Outbound Discarded" },
+            .OutNoRoutes = { .key = "Datagrams Outbound No Route" },
+            .InAddrErrors = { .key = "Datagrams Received Address Errors" },
+            .InHdrErrors = { .key = "Datagrams Received Header Errors" },
+            .InUnknownProtos = { .key = "Datagrams Received Unknown Protocol" },
+            .type = "ipv4",
+            .id = "errors",
+            .family = "errors",
+            .context = "ipv4.errors",
+            .title = "IPv4 TCP errors",

```

Comment:
Why TCP? I think these are just IPv4 error (the same goes for IPv6)

Reply:
**If these are v4 (v6)** TCP packets you name metric `ipv4.tcppackets`/`ipv4.tcperrors` => they go into the "Network->IPv4->Protocol->TCP" section.

Reply:
@stelfrag, please, take a look in this last comment.